### DISCUSSION TITLE
Enabling admins to change their passwords

### DISCUSSION THREAD

**Discussion by jormlm** [globaleaks/globaleaks-whistleblowing-software#3383] [Q&A]
Hello!

From settings it seems you can enable admins to change their passwords. However it looks like that function is disabled in our system. Do you guys know how we can activate that feature? 

Thanks!

![image](https://user-images.githubusercontent.com/123004930/223681158-ec26db8e-ad19-496f-af76-cc3074cbbd71.png)

**Comment by jormlm** [comment]
Can anybody please help? @evilaliv3? 

Question: Can admins change their passwords? It seems there's a setting for it, however it doesn't work in our system.

**Comment by evilaliv3** [comment]
Thank you for your question @jormlm. I'm sorry for the delay in this answer, i just noticed your questions.

From what i see in the screen actually it seems that your system have a major issue.
Is it possible that you have recently removed an old administrator that have setup the system?
This user was the only one with access to key escrows and by removing their account without transfering the keys you have created a special not-managed condition.

**Reply by jormlm** [reply]
No problem, thank you for answering! 

It looks like that's actually what happened. The admin user that configured GlobaLeaks was deleted. Is there anything we can do?

**Reply by jormlm** [reply]
Anything we can do @evilaliv3? Thank you for your time regardless :)
